CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning].
 
 - Previously deprecated `helpers.hpp` header file has been removed.
   [[#410](https://github.com/ethereum/evmc/pull/410)]
+- Prreviously deprecated `EVMC_CONSTANTINOPLE2` and `EVMC_LATEST_REVISION` revisions have been removed.
```

Comment:
```suggestion
- Previously deprecated `EVMC_CONSTANTINOPLE2` and `EVMC_LATEST_REVISION` revisions have been removed.
```

Reply:
Is this discrimination against purrrrring changelogs? :)